python/lsst/ts/m2gui/utility_monitor.py

```diff
@@ -65,10 +72,18 @@ class UtilityMonitor(object):
     displacements : `dict`
         Displacement sensors. The key is the displacement sensor name. The
         value is the reading in the unit of mm.
+    digital_status_input : `int`
```

Comment:
For OpenSpliceDDS/SAL, the policy is not to use bitfields. Instead all values inside bitfield shall be "expanded" into elements in telemetry. So instead of something like:

```
errors.flags # a bitfield, 0x01 = Error 1, 0x02 = Error 2, ..
```

this is used:

```
errors.error1, errors.error2, ...
```

Of course you are on JSON level, but please consider expanding the bitfield into dict fields.

Reply:
This is the default interface in the M2 cell LabVIEW code right now. I would not change it at this moment (that LabVIEW code is kind of spaghetti and I have no plan to change it at all). But after John translated it to C++ code, we should be able to expand the bit value as what you suggested. Thanks!